docs/scripts/buildApi.ts

```diff
@@ -30,8 +30,9 @@ import getStylesCreator from '@material-ui/styles/getStylesCreator';
 import { createMuiTheme } from '@material-ui/core/styles';
 import { getLineFeed, getUnstyledFilename } from './helpers';
 
-// Only run for ButtonBase
+// Only run for the named component
 const TEST = false;
+// const TEST = 'Accordion';
```

Comment:
It seems to me we can remove this and related code entirely in favor of `yarn docs:api --grep ButtonBase` which got introduced in https://github.com/mui-org/material-ui/pull/21731.

Or is you workflow not supported by `--grep`?

Reply:
I had missed that it's now supported by `yarn docs:api`. That works for me! 👍🏼

Reply:
I'll try to remember to mention script changes in slack (as a simple form of a changelog).